spec.bs

```diff
@@ -1437,12 +1437,20 @@ the API {{Window/navigator}}.{{Navigator/deprecatedReplaceInURN()}}, which allow
 macros into the [=fenced frame config/mapped url=] corresponding to a given [=urn uuid=] or
 {{FencedFrameConfig}}.
 
+Note: To help with ease of adoption,
```

Comment:
Does this one have a different plan from deprecatedReplaceInURN()? The note above says until 2026, and here it says third party cookie deprecation. If they have the same plan, we should probably update both, or possibly a single note paragraph to cover both the two APIs?

Reply:
They're different timelines, hence the different wording and different links.